src/frontend/get-github-configuration.ts

```diff
@@ -133,8 +133,7 @@ const removeFailedConnectionsFromDb = async (req: Request, installations: any, j
 };
 
 export default async (req: Request, res: Response, next: NextFunction): Promise<void> => {
-	const { githubToken } = req.session;
-	const { jiraHost } = res.locals;
+	const { jiraHost, githubToken } = res.locals;
```

Comment:
Was `githubToken` moved from `req.session` to `res.locals`? Should we have a feature flag here?

Reply:
Yes.  It's the only way to centralize the check if the github token is valid or not.  We can feature flag it, but it affects many different areas.  I did manually test these in staging as well as had to updated the tests surrounding it to make sure that the res.locals var is set  or else they would fail.